mailpoet/lib/Mailer/Methods/MailPoet.php

```diff
@@ -10,6 +10,7 @@
 use MailPoet\Services\AuthorizedEmailsController;
 use MailPoet\Services\Bridge;
 use MailPoet\Services\Bridge\API;
+use MailPoet\Util\Url as UrlUtil;
```

Comment:
I apologize, but I'm still surprised about this alias because it looks a little bit weird to me.
I tried to check the code in our plugin, and we use an alias `UrlHelper` or no alias.
But it's up to you :) Feel free to skip this.

Reply:
the naming is basically like converting `Util\Url ` to `UrlUtil` i'm fine with your suggestion as well :)